### PR TITLE
remove unused pact code

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by czue** [comment]
nice. :skull: on :white_check_mark:

File: custom/_legacy/pact/regimen.py [hunk 1]
```diff
@@ -1,31 +1,9 @@
...
-#            update_ret['nonartregimen'] = str(len(code_arr)) if len(code_arr) > 0 else ""
-#        digit_strings = ["zero", 'one', 'two', 'three','four']
-#        for x in range(1,5):
-#            prop_prop = prop_fmt % digit_strings[x]
-#            if x > len(code_arr):
-#                update_ret[prop_prop] = ''
-#            else:
-#                update_ret[prop_prop] = str(code_arr[x-1])
-#    return update_ret
+from pact.enums import DAY_SLOTS_BY_TIME, DAY_SLOTS_BY_IDX, DOT_ART, DOT_NONART, CASE_ART_REGIMEN_PROP, CASE_NONART_REGIMEN_PROP
```
Comment (dimagimon) [inline custom/_legacy/pact/regimen.py]: E501 line too long (128 > 115 characters)

**Comment by dannyroberts** [comment]
:white_check_mark: (first failure was just lintreview, which i fixed in last commit)

**Comment by benrudolph** [comment]
:skull: :construction:

**Pull request merged**